src/connector.js

```diff
@@ -599,6 +603,16 @@ function initEngine() {
 			}
 		}
 
+		if ( urlParams.year ) {
+			const year = Number.parseInt( urlParams.year );
+			if ( Number.isInteger( year )  && ( year >= 2000 )  && ( year <= ( new Date().getFullYear() + 1 ) ) ) {
+				aqString += ' @uri=".ca/' + urlParams.year + '"';
+			}
+			else {
+				aqString += ' NOT @uri';
+			}
+		}
+
```

Comment:
@igkislev Please review and make sure that @bblaisATcoveo has implemented the desired outcome as per your discussions. Please confirm here. Thank you.